Detectors/EMCAL/calibration/src/EMCALChannelCalibrator.cxx

```diff
@@ -0,0 +1,123 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "EMCALCalibration/EMCALChannelCalibrator.h"
+#include "Framework/Logger.h"
+#include "CommonUtils/MemFileHelper.h"
+#include "CCDB/CcdbApi.h"
+#include "DetectorsCalibration/Utils.h"
+#include <boost/histogram.hpp>
+#include <boost/histogram/ostream.hpp>
+#include <boost/format.hpp>
+#include <cassert>
+#include <iostream>
+#include <sstream>
+#include <TStopwatch.h>
+
+namespace o2
+{
+namespace emcal
+{
+
+using Slot = o2::calibration::TimeSlot<o2::emcal::EMCALChannelData>;
+using clbUtils = o2::calibration::Utils;
+using boost::histogram::indexed;
+
+//===================================================================
+//_____________________________________________
+void EMCALChannelData::fill(const gsl::span<const o2::emcal::Cell> data)
+{
+  // fill container
```

Comment:
Never put any method documentation into the cxx file, even though this was common practice in AliRoot/AliPhysics and recommended by the offline team. Documentation must always go into the header file. Even though doxygen would allow for having the documentation in the cxx file, but when having it in the header file clever tools like VSCode or Xcode can make use of it providing the users with help texts, which is not the case if the documentation is in the cxx file.